### ISSUE TITLE
Add line wrap to nammu

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#113]
Several more complex atf files have very long lines. We should make sure nammu can correctly wrap long lines to the screen size

**Comment by raquel-ucl** [comment]
One would expect JEditPane has some setting to make it wrap lines, but it doesn't have it. Apparently the line breaking algorithm needs to be implemented from scratch.

http://stackoverflow.com/questions/16641065/word-wrap-in-jeditorpane-and-system-font

**Comment by raquel-ucl** [comment]
I was wrong. JEditPane does automatically wrap line ends but things get complicated when you wrap it on a ScrollPane. 

One of the reasons it didn't work, is because I was forcing the preferredSize attribute on the ScrollPane instead of the container of the JEditorPane+LineNumberPanel. I still don't understand why this change re-enables the line wrapping, but it does.

However, this breaks the line numbers panel, since it is developed in a way where it checks for new line characters `\n` and when a long line is displayed in several lines, the next line number should appear parallel to the line after the really long one. 

This is how it looks:

<img width="690" alt="screen shot 2016-07-05 at 14 24 53" src="https://cloud.githubusercontent.com/assets/10402246/16586161/49ef4748-42bc-11e6-8f13-820934aa2720.png">


This is how it should look:

<img width="690" alt="screen shot 2016-07-05 at 14 23 06" src="https://cloud.githubusercontent.com/assets/10402246/16586128/29ecf06c-42bc-11e6-91a5-6ce5aa1b9272.png">


Thankfully, someone made a [Java class for line numbers](https://tips4java.wordpress.com/2009/05/23/text-component-line-number/) which I have successfully imported locally, so now line wrapping works and line numbers work. However the scroll panel is broken! I need to fix the layout so all works at the same time :)
Also need to import this class via Maven which shouldn't be a problem, but I'm not yet sure how to do it.

**Comment by raquel-ucl** [comment]
This all works now locally, but unfortunately it doesn't work from the JAR :(

I am using the Java class for line numbers I mentioned earlier, which is really nice. To import it locally, I just needed to add compile the java class and put it in my virtualenv's site-packages. This way `import TextLineNumber` will work no problems.

I also changed the layout so the line wrapping and scrolling functionality behaves correctly.

In the JAR, I've tried adding in the copy-resources stage a step to put the `TextLineNumber.class` file in `Lib` since that is in the classpath, however although it's correctly packed (I checked unzipping the JAR and looking inside), it won't be imported properly causing a `ClassNotFoundException`. It doesn't complain when importing it, but it does when using it. I've tried importing it as a jar as well, and putting it in other folders in the class path, but with no luck. I run out of time before the delivery, so I'll keep on working on this on Thursday.
